r-p-s.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="r-p-s.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <h1>Rock, Paper, Scissors Game</h1>
+
+    <p>
```

Comment:
I would change this <p> tag to a <div>. <p> tag stands for paragraph so mostly just used for text, not to contain other elements.